public/assets/ts/admin/page-editor/editor-nodes/EditorNode.tsx

```diff
@@ -21,6 +23,8 @@ export const EditorNode: React.FC<EditorNodeProps & { value: any }> = (
     }
   } else if (props.name.includes("Markdown")) {
     return <EditorMarkdownNode {...props} />;
+  } else if (isTermNumber) {
```

Comment:
I like the thought behind the approach you did! It unfortunately misses a few cases though, like the `year` field on http://localhost:3000/admin/editor?page=documents/budgets.json

thoughts on directly checking the type of the value instead? 👀 
```suggestion
  } else if (typeof value === "number") {
```

Reply:
I can't directly check the type of value because the returned value is a string. But I could do something like this instead. 

```suggestion
  } else if (!isNaN(parseInt(value))) {
```